workbench-core/authentication/src/plugins/cognitoUserManagementPlugin.ts

```diff
@@ -34,26 +20,17 @@ import { UserManagementPlugin } from '../userManagementPlugin';
  */
 export class CognitoUserManagementPlugin implements UserManagementPlugin {
   private _userPoolId: string;
-
-  private _cognitoClient: CognitoIdentityProviderClient;
+  private _aws: AwsService;
 
   /**
    *
    * @param userPoolId - the user pool id to update with the plugin
+   * @param aws - a {@link AwsService} instance with permissions to perform Cognito actions
    */
-  public constructor(userPoolId: string) {
+  public constructor(userPoolId: string, aws: AwsService) {
```

Comment:
Breaking change? Are you sure you've hit all of the callers in our repo?
Please add a note to the Readme that indicates the version number rev where the breaking change occurs and the course of action to be taken to fix it.